### DISCUSSION TITLE
Can’t attach LibreChat’s own `deploy-compose.yml` or `docker-compose.yml`.

### DISCUSSION THREAD

**Discussion by ath** [danny-avila/LibreChat#4584] [Troubleshooting]
### What happened?

This is about attaching LibreChat’s `deploy-compose.yml` and `docker-compose.yml` files that ship with the repo.

I tried to attach both of the files but it resulted in an error message:
_Unable to determine file type for: deploy-compose.yml_

Renaming it to `.yaml` doesn’t solve it.
Renaming it to `.txt` allows me to first attach it, for a moment, only to then be confronted with the message: _Error processing file_

I tried this in Firefox `131.0.3` and Chrome `130.0.6723.92 (Offizieller Build) (64-Bit)`, both on Windows 11 Pro (german).

### Steps to Reproduce

Start a fresh chat session, for example with Anthropic Sonnet 3.5.
Attach the `deploy-compose.yml` that is included in the official LibreChat repo.

### What browsers are you seeing the problem on?

Firefox, Chrome

### Screenshots

![grafik](https://github.com/user-attachments/assets/7b6d17b1-dca8-46bd-a4bb-3cf11e7b2b3f)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
You are better off copying the content as it is short enough, and YAML is not inherently supported by the RAG API in any case.

The handling of these files will also be addressed by this: https://github.com/danny-avila/LibreChat/issues/2755

**Reply by ath** [reply]
Sure, there are workarounds. It’s true however that some users would expect that they can post any text file type, as they can when using ChatGPT or Claude.

The correct solution is imo that we add this functionality.
